app/src/main/java/org/mozilla/focus/MainActivity.kt

```diff
@@ -76,7 +76,7 @@ class MainActivity : LocaleAwareAppCompatActivity(), OnUrlEnteredListener {
     }
 
     private fun onValidBrowserIntent(url: String, source: Source) {
-        sessionManager.createSession(source, url)
+        ScreenController.showBrowserScreenForUrl(supportFragmentManager, url, source)
```

Comment:
lol - I was expecting a super-long last commit, but this is pretty good (albeit anticlimactic).